server/graphql/common/expenses.ts

```diff
@@ -297,33 +335,7 @@ export const canVerifyDraftExpense: ExpensePermissionEvaluator = async (req, exp
   return remoteUserMeetsOneCondition(req, expense, [isOwner, isCollectiveAdmin, isHostAdmin]);
 };
 
-/**
- * Returns the list of items for this expense.
- */
-export const getExpenseItems = async (expenseId: number, req: express.Request): Promise<ExpenseItem[]> => {
```

Comment:
Removed this one as it was lost in the middle of the permissions checks and it seems just as simple to call the loader directly